app/services/idv/analytics_events_enhancer.rb

```diff
@@ -1,112 +1,91 @@
-      idv_in_person_locations_searched
-      idv_in_person_prepare_submitted
-      idv_in_person_prepare_visited
-      idv_in_person_proofing_address_visited
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_completed
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_ingestion_error
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_started
-      idv_in_person_proofing_nontransliterable_characters_submitted
-      idv_in_person_proofing_residential_address_submitted
-      idv_in_person_proofing_state_id_submitted
-      idv_in_person_proofing_state_id_visited
-      idv_in_person_ready_to_verify_sp_link_clicked
-      idv_in_person_ready_to_verify_what_to_bring_link_clicked
-      idv_in_person_send_proofing_notification_attempted
-      idv_in_person_send_proofing_notification_job_completed
-      idv_in_person_send_proofing_notification_job_exception
-      idv_in_person_send_proofing_notification_job_skipped
-      idv_in_person_send_proofing_notification_job_started
-      idv_in_person_switch_back_submitted
-      idv_in_person_switch_back_visited
-      idv_in_person_usps_proofing_enrollment_code_email_received
-      idv_in_person_usps_proofing_results_job_completed
-      idv_in_person_usps_proofing_results_job_deadline_passed_email_exception
-      idv_in_person_usps_proofing_results_job_deadline_passed_email_initiated
-      idv_in_person_usps_proofing_results_job_email_initiated
-      idv_in_person_usps_proofing_results_job_enrollment_incomplete
-      idv_in_person_usps_proofing_results_job_enrollment_updated
-      idv_in_person_usps_proofing_results_job_exception
-      idv_in_person_usps_proofing_results_job_please_call_email_initiated
-      idv_in_person_usps_proofing_results_job_started
-      idv_in_person_usps_proofing_results_job_unexpected_response
-      idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review
-      idv_in_person_usps_request_enroll_exception
-      idv_ipp_deactivated_for_never_visiting_post_office
-      idv_link_sent_capture_doc_polling_complete
-      idv_link_sent_capture_doc_polling_started
-      idv_mail_only_warning_visited
-      idv_native_camera_forced
-      idv_not_verified_visited
-      idv_phone_use_different
-      idv_request_letter_visited
-      idv_sdk_selfie_image_capture_closed_without_photo
-      idv_sdk_selfie_image_capture_failed
-      idv_sdk_selfie_image_capture_opened
-      idv_selfie_image_added
-      idv_session_error_visited
-      idv_threatmetrix_response_body
-      idv_usps_auth_token_refresh_job_completed
-      idv_usps_auth_token_refresh_job_network_error
-      idv_usps_auth_token_refresh_job_started
-      idv_verify_by_mail_enter_code_submitted
-      idv_verify_by_mail_enter_code_visited
-      idv_verify_in_person_troubleshooting_option_clicked
-      idv_warning_action_triggered
-      idv_warning_shown
-    ].to_set.freeze
+    EXCLUDED_FRONTEND_EVENT_METHODS = [
```

Comment:
@aduth - We exclude these events and just want to confirm that this list makes sense. Some events like `:idv_personal_key_downloaded` are _not_ ignored since there is an expectation for `Idv::Session` information to present at this point.

Reply:
Frankly, this is carryover from the original list as I was not confident that we have the user and Idv::Session present for all the events here.

Upon further reflection on the FrontendLogger/FrontendLogController, having them be enhanced shouldn't cause any issues.